travvy/src/components/LogIn.vue

```diff
@@ -1,36 +1,70 @@
 
 <template>  
-
   <div class="LogIn">
         <h2>Log In to TRAVVY</h2>
         <form>
         <!-- this form takes the sign up input from users - binds to the user form --> 
           <label>Email: </label>
-          <input type="text" v-bind="UserForm"/> 
+          <input type="text" v-model="email" id="email"/> 
           <br>
           <br>
           <label>Password: </label>
-          <input type="password" v-bind="UserForm"/> 
+          <input type="password" v-model="password" id="password"/> 
           <br>
+          <!-- 
+            this was causing a ton of problems
           <input type="checkbox" @click="showPassword">Show Password
+          --> 
         </form>
         <br>
         <br>
-        <button class="search"><span>Log In</span></button>
+        <button class="search" @click='login'>Log In</button>
         
     </div>
 </template>
 
 <script>
-export default {
-  name: 'LogIn',
-  props: {
-    msg: String
+  //import from the AuthenticationService file
+  import AuthenticationService from '../services/AuthenticationService.js'
+
+  export default {
+    data(){
+      return {
+        email: '',
+        password: '',
+        error: null
+      }
+    }, 
+    methods:{
+      //the login method, this is what runs when the log in button is pushed
+      async login(){
```

Comment:
great job with incorporating this functionality, we should now be able to create users and log in as them successfully